File: Desafios_Java/Solucionando_Desafios_Matemáticos_em_Java/desafio_01.java
/* Desafio 1 - Cálculo de viagem*/

package Desafios_Java.Solucionando_Desafios_Matemáticos_em_Java;

import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;
import java.util.Arrays;
import java.util.List;
import java.util.stream.Collectors;

public class desafio_01 {
    public static void main(String[] args) throws IOException {

        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));

        List<Double> param = Arrays.asList(reader.readLine().split(" "))
                .stream()
                .map(value -> Double.parseDouble(value))
                .collect(Collectors.toList());

        double consume = (param.get(0) * param.get(1)) / 12;

        System.out.format("%10.3f%n", consume);
    }
}

File: Desafios_C_Sharp/Solução_de_Problemas_Básicos_ em_C_Sharp/desafio_01.cs
/* Desafio 1 - Visita na Feira*/

using System; 

class desafio_01
{
    static void Main(string[] args)
    { 
      int a, b, x;
      string[] input = Console.ReadLine().split(' ');
      
      a = Convert.ToInt32(input[0]);
      b = Convert.ToInt32(input[1]);

      x = a + b;

      Console.WriteLine("X = {0}", x);

    }

}
